app/src/organisms/Desktop/Devices/ProtocolRun/ProtocolRunCamera/ImageGalleryContainer/GalleryItemCard.tsx

```diff
@@ -53,19 +47,25 @@ export function GalleryItemCard(props: GalleryItemCardProps): JSX.Element {
       total: totalStepStr,
     })
   }
+  const state = isSkeleton ? 'loading' : null
+  const centerPrimaryText = t('step_command', {
```

Comment:
```suggestion
  const stepCommandText = t('step_command', {
```
Nit, because we are working outside of the API of the `MediaContainerContent` component here, the variable name should be ideally reflective of how it's used in this internals of `GalleryItemCard`.